src/styles/main.scss

```diff
@@ -1,3 +1,81 @@
+@import 'variables';
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
```

Comment:
better use min-height: 100vh

`Using min-height: 100vh; allows for flexible content display, adapting to varying viewport sizes and accommodating dynamic content. It prevents overflow issues and enhances accessibility by ensuring all users can access content without encountering layout problems. Additionally, it future-proofs your design, allowing for seamless adjustments as your website evolves without the need for manual height adjustments.`